### PR TITLE
finished the modal lightbox

### PR DESCRIPTION
**Pull request by dylanqian29**
Hi, @zhujohnny! Just finished the lightbox modal, can you help me check the code

### CODE REVIEW

File: client/src/components/Photo.jsx [hunk 1]
```diff
@@ -25,13 +25,13 @@ export default class Photo extends React.Component {
         if (this.state.hovered === true) {
             return (
             <span class = "square" onMouseEnter={() => this.mouseOnPic()} onMouseLeave={() => this.mousLeavePic()}>
-                <img height="250" width = "250" src={`${this.props.url}`} className="image" />
```
Comment (zhujohnny) [inline client/src/components/Photo.jsx]: Check spelling for mousLeavePic (add an 'e'?)

File: client/src/components/Photo.jsx [hunk 2]
```diff
@@ -25,13 +25,13 @@ export default class Photo extends React.Component {
...
+                <a href={`${this.props.url}`} data-lightbox = "food" ><img height="250" width = "250" src={`${this.props.url}`} className="image" /></a>
             </span>
```
Comment (zhujohnny) [inline client/src/components/Photo.jsx]: Is there a reason you're using a template literal for {`{this.props.url}`}? Also, not immediately obvious to me how data-lightbox is used or what "food" is.

File: client/src/components/Photo.jsx (hunk 1 above)
Comment (dylanqian29) [reply client/src/components/Photo.jsx]: oh yea you are right, ill check it now! Thanks!

File: client/src/components/Photo.jsx (hunk 2 above)
Comment (dylanqian29) [reply client/src/components/Photo.jsx]: you are right, I changed to {this.props.url} now. 

lightbox is a package that to make the pop out modal, and that's just the way to tell the package where the css should be for.  and if all the pictures have the same value for data-lightbox, they will be shown as serial, and that's what I did in this part

**Comment by dylanqian29** [comment]
@zhujohnny I just made the change, can you check again before merging. Thanks!

**Pull request merged**